jep-014-string-functions.md

```diff
@@ -21,7 +21,7 @@ int find_first(string $subject, string $sub[, int $start[, int $end]])
 ```
 Given the `$subject` string, `find_first()` returns the zero-based index of the first occurence where the `$sub` substring appears in `$subject` or `null` if it does not appear.
 
-The `$start` and `$end` parameters are optional and allow restricting the range within `$subject` in which `$sub` must be found.
+The `$start` and `$end` parameters are optional and allow restricting to the slice `[$start:$end]` the range within `$subject` in which `$sub` must be found.
```

Comment:
There should be some examples demonstrating the significance of this interpretation, e.g.
* ```find_first('subject string', 's', `-6`)``` results in `8`, because the search is restricted to slice `[-6:]` (i.e., the final "string").
* ```find_first('subject string', 'string', `8`, `11`)``` results in `null`, because the search is restricted to slice `[-6:11]` (i.e., "str", which does not contain "string").

Reply:
Agreed. Thanks for pointing this out.